TombEngine/Objects/TR3/Object/tr3_boss_object.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include "Game/items.h"
+
+namespace TEN::Entities::Object::TR3
+{
+	enum BossItemFlags
+	{
+		BOSSFlag_Object = 0, // BossFlagValue enum
+		BOSSFlag_Rotation = 1, // Store rotation for use, like puna when summoning.
+		BOSSFlag_ShieldIsEnabled = 2,
+		BOSSFlag_AttackType = 3, // Which attack was performed.
+		BOSSFlag_AttackCount = 4, // Will change behaviour after some attack.
+		BOSSFlag_DeathCount = 5,
+		BOSSFlag_ItemNumber = 6, // To check if summon is dead.
+		BOSSFlag_ExplodeCount = 7
+	};
+
+	enum BossFlagValue
+	{
+		BOSS_ShockwaveRing      = (1 << 0),
+		BOSS_ShockwaveExplosion = (1 << 1),
+		BOSS_Shield             = (1 << 2),
+		BOSS_Lizard             = (1 << 3)
+	};
+
+	extern void BOSS_EffectShieldControl(int itemNumber);
```

Comment:
Remove `BOSS_` prefixes from all functions. We are not using this kind of notation in TEN codebase.

Reply:
Fixed, added a namespace for them:
TEN::Entities::Object::TR3::Boss